packages/client/src/api/wargames_api/index.ts

```diff
@@ -503,11 +505,31 @@ export const deleteForce = (dbName: string, forceId: string): Promise<Wargame> =
     const updatedData = newDoc.data
     const forces = updatedData.forces.forces
     updatedData.forces.forces = forces.filter((force: ForceData) => force.uniqid !== forceId)
-    updatedData.channels.complete = calcComplete(forces)
+    updatedData.channels.channels.forEach((v) => (v.participants as CoreParticipant[] | ForceData[]) = forces)
```

Comment:
Can you explain what the above line is doing?

`forces` is the full list of forces in the game.

We don't put the full list of forces into the `participants` of every channel.

Reply:
The participant assigns a value to each channel each time, at the moment it can be seen in the console that it is also transferred to the mapping
![mapping participant](https://user-images.githubusercontent.com/20753599/160540688-a083e85e-9acb-4b5e-98d5-098c74e415fd.PNG)